tests/support/parser/__init__.py

```diff
@@ -306,7 +306,7 @@ def parse_args(self, args=None, values=None):
             with open(self.options.names_file, 'rb') as fp_:
                 lines = []
                 for line in fp_.readlines():
-                    lines.append(line.strip())
+                    lines.append(line.strip().decode())
```

Comment:
First of all, you can only do this, *if* on PY3.
Second, you decode using the `__salt_system_encoding__`.
This happens to just work because there are only ASCII characters on the file...